.github/workflows/linux_builds.yml

```diff
@@ -4,79 +4,88 @@ on: [push, pull_request]
 # Global Settings
 env:
   GODOT_BASE_BRANCH: master
-  SCONSFLAGS: platform=linuxbsd verbose=yes warnings=extra werror=yes debug_symbols=no --jobs=2 module_text_server_fb_enabled=yes
-  SCONS_CACHE_LIMIT: 4096
+  SCONSFLAGS: verbose=yes warnings=extra werror=yes debug_symbols=no module_text_server_fb_enabled=yes
 
 concurrency:
   group: ci-${{github.actor}}-${{github.head_ref || github.run_number}}-${{github.ref}}-linux
   cancel-in-progress: true
 
 jobs:
-  linux-editor:
+  build-linux:
     runs-on: "ubuntu-20.04"
-    name: Editor (target=release_debug, tools=yes, tests=yes)
+    name: 🐧 Linux Builds
```

Comment:
Right now the jobs show up with duplicate names. We may want to just leave this blank (the one on the right), since the name is already defined at the top of the file (the one on the left).

![ci](https://user-images.githubusercontent.com/1646875/131954147-56cd0024-47fd-45f8-b24c-2b3ca80c694e.png)

Reply:
Removing the name results in the job id being printed. Not sure if that's better:
![better](https://user-images.githubusercontent.com/1687918/131975996-53bb04ed-311c-4d27-bf90-f0b501d5565c.png)

Reply:
I can try with an empty (or single space) name

Reply:
Empty string still prints the job id

Reply:
using a string with a single space seems to work.

Reply:
Updated to show the names like before:

![matrixnames](https://user-images.githubusercontent.com/1687918/133225520-e9d285f5-3ee4-4848-b3db-45f4286c2964.png)